Tasks/NuGetAuthenticateV0/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 0,
         "Minor": 156,
-        "Patch": 1
+        "Patch": 3
```

Comment:
@aasim Looks like your maven PR (which bumped the versions) wasn't cherry-picked to 156 yet?

Reply:
Its in review right now, I have to port your changes to the merge as well, and probably bump these task versions too.